dll/win32/shell32/CShellLink.h

```diff
@@ -98,12 +98,15 @@ class CShellLink :
     HRESULT SetAdvertiseInfo(LPCWSTR str);
     HRESULT WriteAdvertiseInfo(LPCWSTR string, DWORD dwSig);
     HRESULT SetTargetFromPIDLOrPath(LPCITEMIDLIST pidl, LPCWSTR pszFile);
+    HICON CreateShortcutIcon(LPCWSTR wszIconPath, INT IconIndex);
 
 public:
     CShellLink();
     ~CShellLink();
     static INT_PTR CALLBACK SH_ShellLinkDlgProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
 
+    BOOL m_bInitializing;
```

Comment:
This shouldn't need to be public. Please move it up with the other member variables.

Reply:
Made it private.